packages/ag-charts-types/src/chart/tooltipOptions.ts

```diff
@@ -61,7 +61,7 @@ export interface AgTooltipRendererResult {
     class?: CssColor;
 }
 
-export interface AgSeriesTooltipRendererParams<TDatum> extends AgChartCallbackParams<TDatum> {
+export interface AgSeriesTooltipRendererParams<TDatum = any> extends AgChartCallbackParams<TDatum> {
```

Comment:
Ditto?

Reply:
As above, to align with the grid convention for callbacks that can optionally be untyped

Reply:
Reverted